index.html

```diff
@@ -19,29 +19,39 @@
                 <li>
                     <img width="" src="/images/assets/Twitter logo/PNG/2021 Twitter logo - blue.png" alt="Twitter Logo" />
                 </li>
+
                 <li>
```

Comment:
Wrap the image tag and a-tag-title in their own div, it is easier for styling, positioning and both will act as links